tools/cfngen/gluecf/schema.go

```
@@ -62,6 +66,9 @@ func inferJSONColumns(t reflect.Type, customMappingsTable map[string]string) (co
 			if skip {
 				continue
 			}
+			if len(comment) > maxCommentLength { // clip
```

1. I am thinking it is probably better to throw an exception here? That will let the person that wrote the description that they have to re-write the description to something that fits the CFN. By truncating it, we might be leaving out the most important bits that the author was trying to convey. 
2. As I see, all parsers have description now, right? Might worth throwing an exception if some field doesn't have one, as a way to make sure this is enforced.

---

I thought about 1) but I want to allow long lines because we will also use these tags in a future API where we build schemas. I don't think the truncation is a problem as it happens rarely and only for really verbose descriptions. I hit this edge case because I was simply cut-n-pasting the AWS docs for the fields into the descriptions (some are very long). I'd like to let the developers just mindlessly do this kind of copying and not worry about the length.

Suggestion 2) is a really good idea. I've implemented it with a nice error message telling you the struct and fields that is missing the description. This will force devs to not be lazy and always document the the log parsers.

---

Sound fair.